R/reporter-tap.R

```diff
@@ -29,20 +29,21 @@ TapReporter <- R6::R6Class("TapReporter", inherit = Reporter,
       if (!self$has_tests)
         return()
 
-      cat("1..", self$n, '\n', sep = '')
+      self$cat_line("1..", self$n)
       for (i in 1:self$n) {
         if (!is.na(self$contexts[i])) {
-          cat("# Context", self$contexts[i], "\n")
+          self$cat_line("# Context ", self$contexts[i], " ")
```

Comment:
Trailing whitespace isn't required by TAP.

On Tue, Mar 22, 2016 at 6:41 PM Kirill Müller <notifications@github.com>
wrote:

> In R/reporter-tap.R
> <https://github.com/hadley/testthat/pull/420#discussion_r57089060>:
>
> >        for (i in 1:self$n) {
> >          if (!is.na(self$contexts[i])) {
> > -          cat("# Context", self$contexts[i], "\n")
> > +          self$cat_line("# Context ", self$contexts[i], " ")
>
> @dkesh <https://github.com/dkesh>: I'm reworking output for the TAP
> reporter. Does it require trailing whitespace, or would it work without
> them?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hadley/testthat/pull/420/files/c1146de2c3ca0d721fb7e3cf3ce3ade9e2a9f81f#r57089060>
>